packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -108,7 +109,14 @@ export class GroupHeaderBase extends BaseComponent<IGroupHeaderProps, IGroupHead
           <div className={this._classNames.dropIcon}>
             <Icon iconName="Tag" />
           </div>
-          <button type="button" className={this._classNames.expand} onClick={this._onToggleCollapse} {...expandButtonProps}>
+          <button
+            type="button"
+            className={this._classNames.expand}
+            onClick={this._onToggleCollapse}
+            aria-expanded={group ? !group.isCollapsed : undefined}
+            aria-controls={group && !group.isCollapsed ? groupedListId : undefined}
```

Comment:
I'm not sure that we should remove this when `group && group.isCollapsed` looking at the w3 examples:

https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region

Perhaps it should be:

```tsx
aria-controls={group ? groupedListId : undefined}
```